src/models/app_configuration.js

```diff
@@ -82,6 +82,20 @@ function findApp (config, alias) {
   return findDefaultApp(config);
 }
 
+function checkAlreadyLinked (apps, name, alias) {
+
+  const appAliasExists = apps.some(app => alias && app.alias === alias);
```

Comment:
nitpick: we try to avoid implicit null test and always use `!= null`. Can add it?

```
  const appAliasExists = apps.some((app) => alias != null && app.alias === alias);
```